css/style.css

```diff
@@ -1,18 +1,522 @@
-.partners-list {
-	width: 100%;
-	margin-bottom: 20px;
-	border: 1px solid #000000;
-	border-collapse: collapse; 
-	background-color: #7de0d3;
+
+body{
+	font-family: "Bakbak One", "Roboto", sans-serif;
+	font-weight: 400;
+	font-style: normal;
+	color: #fff;
+	background-color: #1E1E1E;
+}
+
+a{
+  text-decoration: none;
+  color: currentColor;
+  text-transform: uppercase;
+}
+
+ul,
+ol{
+  list-style: none;
+  color: currentColor;
```

Comment:
you don't need to add the current color to all tags - this property is inherited from the parent